specification/security/resource-manager/Microsoft.Security/preview/2023-06-01-preview/aggregations.json

```diff
@@ -0,0 +1,193 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Aggregations Query",
+    "description": "API spec for Microsoft.Security (Azure Security Center) Aggregations Query",
+    "version": "2023-06-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Security/aggregations": {
+      "post": {
+        "tags": [
+          "aggregations"
+        ],
+        "description": "Run pre-defined aggregation query against Microsoft.Security data types for scopes specified in the request.",
+        "operationId": "Aggregations",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "name": "aggregations",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AggregationRequest"
+            },
+            "description": "Request specifying Aggregation type and scopes."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Result of the aggregation query operation",
+            "schema": {
+              "$ref": "#/definitions/AggregationResponse"
+            }
+          },
+          "default": {
+            "description": "An error occurred while processing the request. See the error.code parameter to identify the specific error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Basic query": {
+            "$ref": "./examples/BasicResult_example.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AggregationRequest": {
+      "description": "Aggregation query type and scopes.",
+      "type": "object",
+      "properties": {
+        "scopes": {
+          "description": "The scopes in which the query will be executed - List of subscription IDs for Azure environments, list of cloud connector IDs for AWS environments and list of GCP connector IDs for GCP environments.",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "Scope."
+          }
+        },
```

Comment:
I wonder... Does it make the API harder or easier to use to mix all the scope ids as strings in one array. I imagine you basically would have to know how to parse all the different kinds, on the library caller side, to work with them and understand which type of scopes they are etc.
Or on any other processor you want to inegrate with this data... (like if ARM had to understand subscription ids in the request, it might be difficult to get it to understand them in the mixed data set)